### ISSUE TITLE
do we want pages for content --> topic?

### ISSUE THREAD

**Issue by pH14** [pH14/hearpetespeak#14]
Right now we have

```
topic 1:
  interview 1 + link
  interview 2 + link

topic 2:
  interview 2 + link
```

Given the same data files, we could also create pages indexed as:

```
interview 1:
  topic 1

interview 2:
  topic 1
  topic 2
```

Is this worthwhile? What would the URL structure look like, can we leverage Hugo to make this easily?

**Comment by jrittenh** [comment]
no filter:

    interview 1
    interview 2
    interview 3

filter:

    interview 1 + timestamp
    interview 2 + timestamp
    interview 3 + timestamp

I'm not familiar at all with Hugo, so I don't know if this is possible, but it would allow for a much more fluid site.

**Comment by pH14** [comment]
Mm if I understand correctly, using the selecting categories to filter down the set of interviews / topics?

**Comment by jrittenh** [comment]
Yeah, basically.  The base URL would link to the full interview, and then the filtered views would link to relevant timestamps for the topic.
